invoice.go

```diff
@@ -22,6 +22,10 @@ type InvoiceParams struct {
 	SubTrialEnd          int64
 	TaxPercent           float64
 	TaxPercentZero       bool
+	Billing string
```

Comment:
Would you mind defining a separate data type for `Billing`? A little like this:
https://github.com/stripe/stripe-go/blob/master/sub.go#L5,L7